nas/dir.go

```diff
@@ -0,0 +1,64 @@
+/*
+Package nas provides support for efficiently working with
+network attached storage (NAS).
+*/
+package nas
```

Comment:
The NAS (Network Attached Storage) package provides functions optimized to work with mounted storage volumes.
Moved from the `tackle2-addon` (lib) repository.